src/Microsoft.TemplateEngine.Edge/Template/TemplateCreator.cs

```diff
@@ -56,14 +58,13 @@ public async Task<TemplateCreationResult> InstantiateAsync(ITemplateInfo templat
                 ICreationResult creationResult = null;
                 string targetDir = outputPath ?? _environmentSettings.Host.FileSystem.GetCurrentDirectory();
 
+                Timing contentGeneratorBlock = Timing.Over(_logger, "Template content generation");
```

Comment:
nit: Why not `using`?

Reply:
not to break indentation - previous version used using and made it hard to review.